src/main/views/user-details.njk

```diff
@@ -141,7 +141,8 @@
     <input type="hidden" name="_userId" value="{{ content.user.id }}">
     <div class="govuk-button-group">
       {{ govukButton({ text: "Edit user", name: "_action", value: "edit", classes: "govuk-button" }) }}
-      {{ govukButton({ text: "Suspend user", name: "_action", value: "suspend", classes: "govuk-button--secondary" }) }}
+      {{ govukButton({ text: "Suspend user", name: "_action", value: "suspend", classes: "govuk-button--secondary" }) if content.user.active }}
+      {{ govukButton({ text: "Un-suspend user", name: "_action", value: "un-suspend", classes: "govuk-button--secondary" }) if not content.user.active }}
```

Comment:
The condition for showing the unsuspend button "if not content.user.active"....does it handle the case where the user is "Archived"? I presume we don't want to show the button is the user status is "Archived"

Reply:
All buttons now hidden when user stale in https://github.com/hmcts/idam-user-dashboard/pull/238/commits/5ff3131225f0041e109daed3e25a65304b8a7b34